### ISSUE TITLE
Black screen but sound

### ISSUE THREAD

**Issue by NicholasB85** [ClusterM/hakchi2#848]
After updating hakchi and adding a couple more games. my snes mini now only shows a black screen when connected to my tv. I have tried flashing back to the original kernal and that does not seen to help. The background music plays and when I shut off the console it says shutting down on the screen... anyone have any ideas on how to fix this issue? did I brick my console?

**Comment by Darky2k1** [comment]
Did you change the Firmware to a different country?

**Comment by djritz0** [comment]
I have this issue too. The picture will appear after about 30 seconds. It’s annoying but will still work

**Comment by NicholasB85** [comment]
I did not change the firmware to another country. I ended up searching through the threads till I found someone who had posted a stock version of their kernal. I flashed that and it worked again. I can only assume my kernal had become corrupt and when I flashed a custom kernal it was reading part of the original kernal. I have since added the custom kernal and retroarch. So far all the games that previously worked are working. I lost half of my saved games I backed up due to whatever happened.
